<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FITSHeaderEditNewKeywordDialog</class>
 <widget class="QWidget" name="FITSHeaderEditNewKeywordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>159</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="leKey">
     <property name="placeholderText">
      <string>Enter search term here</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leComment">
     <property name="placeholderText">
      <string>Enter search term here</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leValue">
     <property name="placeholderText">
      <string>Enter search term here</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lValue">
     <property name="text">
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lKey">
     <property name="text">
      <string>Key:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leKey</tabstop>
  <tabstop>leValue</tabstop>
  <tabstop>leComment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
